lib/improver/nbhood.py

```diff
@@ -39,69 +39,51 @@
     concatenate_cubes)
 
 
-class BasicNeighbourhoodProcessing(object):
-    """
-    Apply a neigbourhood processing kernel to a thresholded cube.
-
-    When applied to a thresholded probabilistic cube, it acts like a
-    low-pass filter which reduces noisiness in the probabilities.
-
-    The kernel will presently only work with projections in which the
-    x grid point spacing and y grid point spacing are constant over the
-    entire domain, such as the UK national grid projection.
-
-    A maximum kernel radius of 500 grid cells is imposed in order to
-    avoid computational ineffiency and possible memory errors.
+class Utilities(object):
 
     """
+    Utilities for neighbourhood processing.
+    """
```

Comment:
I think with a bit more hindsight, I agree. I have been looking at the html documentation that is now being produced and this is rather better than including a list in the docstring.